src/redux/reducers/imgDailyReducer.js

```diff
@@ -1,7 +1,7 @@
 import { FETCH_IMAGES_SUCCESS, IS_LOADING } from '../types'
 
 const initialState = {
-  image: {},
+  image: null,
```

Comment:
is image an object? I think you're storing only Image URL which is a string.
If thats a string the default value should be a string as well, I mean an empty string